OmniBLE/OmnipodCommon/Pod.swift

```diff
@@ -79,7 +79,7 @@ public struct Pod {
     // Checked to verify it agrees with value returned by pod during the pairing process.
     public static let cannulaInsertionUnits = 0.5
 
-    public static let cannulaInsertionUnitsExtra = 0.0 // edit to add a fixed additional amount of insulin during cannula insertion
+    public static let cannulaInsertionUnitsExtra = 0.5 // edit to add a fixed additional amount of insulin during cannula insertion
```

Comment:
Unintentional?

Reply:
Unintentional... do I need to correct and re-commit or can you reject just that?

Reply:
Fixed